config/config.json

```diff
@@ -41,6 +41,7 @@
       "provider": "arangodb",
       "username": "root",
       "password": "root",
```

Comment:
Should we remove password from config?

Reply:
I don't think we should, at least for backward compatibility. In case a user uses their own database they don't need a separate file and can use the config. Also, this enables us to use basic auth for non-production environments.